package.json

```diff
@@ -26,6 +26,7 @@
   ],
   "homepage": "https://github.com/amilajack/eslint-plugin-compat#readme",
   "scripts": {
+    "benchmarks": "cross-env NODE_ENV=production babel test/benchmarks.ts --out-dir benchmarks-tmp --source-maps inline --extensions '.ts' && node -r source-map-support/register benchmarks-tmp/benchmarks.js",
     "build": "cross-env NODE_ENV=production rm -rf lib && babel src --out-dir lib --source-maps inline --extensions '.ts'",
     "lint": "eslint --ignore-path .gitignore --ext .js,.ts .",
     "spec": "jest",
```

Comment:
call `benchmarks` subcommand from `test`  subcommand so we can run benchmarks as e2e tests

Reply:
Benchmarks use `class ESLint` which only exists on `eslint^7`. Might make it harder to conditionally handle this in CI.

Reply:
Makes sense